README.md

```diff
@@ -216,7 +223,13 @@ concatenation of the round number treated as a `uint64` and the previous
 signature. At the moment, we are only using BLS signatures on the bls12-381 curves
 and the signature is made over G1.
 
+
+(Note that this requires exposed drand nodes available to query public randomness,
+for instance this won't work with the current League of Entropy nodes, since they
+are not exposing their GRPC endpoints.)
```

Comment:
```suggestion
(Note that this command expects access to a drand group member,
this won't work with the current League of Entropy nodes, since they
are not exposing their GRPC endpoints directly.)
```